x-pack/plugins/fleet/server/routes/data_streams/handlers.ts

```diff
@@ -137,9 +129,8 @@ export const getListHandler: RequestHandler = async (context, request, response)
         serviceDetails: null,
       };
 
-      const { maxIngested, namespace, dataset, type, serviceNames, environments } = useTermsEnum
-        ? await getMetadataFromTermsEnum({ dataStreamName: dataStream.name, esClient })
-        : await getMetadataFromAggregations({ dataStreamName: dataStream.name, esClient });
+      const { maxIngested, namespace, dataset, type, serviceNames, environments } =
+        await getMetadataFromTermsEnum({ dataStreamName: dataStream.name, esClient });
```

Comment:
nit should we rename it `getDatastreamsMetadata`? the form term enum is probably just an implementation details when you consume it

Reply:
Good call. Renamed in https://github.com/elastic/kibana/pull/133634/commits/8728bb3c18bd725b38f13c263110753b97187926